tests/tests/test_boot_basic.py

```diff
@@ -37,3 +37,18 @@ def test_guest_boot():
     m.check_exec('ls -la /sys/firmware/acpi/tables/CCEL')
 
     qm.stop()
+
+def test_early_printk():
+    """
+    Test Early Printk with Debug Off (Intel Case ID 018)
+    """
+
+    qm = Qemu.QemuMachine()
+    qm.qcmd.plugins['boot'].kernel = "/boot/vmlinuz"
```

Comment:
Here we are using the host kernel that can be different from the guest kernel (that is inside the image). I dont think we should follow "stricto sensu" the instructions from Intel. The idea here is to only add `earlyprintk=ttyS0,115200` to kernel commandline

Reply:
You can't add earlyprintk without kernel.  That is what I tried first.

Reply:
ah, I see that it is not a trivial problem, let do it in a following PR and have some time to think about it, it will allow us to wrap up this PR quickly